src/js/components/PageHeader/stories/Simple.js

```diff
@@ -0,0 +1,69 @@
+import React, { useContext } from 'react';
+
+import {
+  Anchor,
+  Box,
+  Button,
+  Menu,
+  PageHeader,
+  Page,
+  PageContent,
+  ResponsiveContext,
+} from 'grommet';
+import { More } from 'grommet-icons';
+
+const actions = [
+  {
+    label: 'Get Started',
+  },
+  {
+    label: 'Follow',
+  },
+  {
+    label: 'File Issue',
+  },
+];
+
+export const Simple = () => {
+  const size = useContext(ResponsiveContext);
+
+  return (
+    // Uncomment <Grommet> lines when using outside of storybook
+    // <Grommet theme={...}>
+    <Page>
+      <PageContent>
+        <PageHeader
+          title="Grommet"
+          subtitle={`Grommet helps you build responsive and accessible 
+          mobile-first projects for the web with an easy to use component 
+          library.`}
+          actions={
```

Comment:
Can we have the Simple story use just a single Button in the `actions`, since I think that's the predominant case?

Reply:
done.